howso/engine/trainee.py

```diff
@@ -3658,21 +3680,25 @@ def _create(
             The library type of the Trainee.
         max_wait_time : int or float, optional
             The maximum time to wait for the trainee to be created.
-        resources : TraineeResources or map of str -> any, optional
+        resources : map of str -> any, optional
             The resources to provision for the trainee.
         overwrite : bool, default False
             If True, will overwrite an existing trainee with the same name.
         """
         if not self.id:
-            trainee = BaseTrainee(**self.to_dict())
             new_trainee = None
             if isinstance(self.client, AbstractHowsoClient):
                 new_trainee = self.client.create_trainee(
-                    trainee=trainee,
+                    name=self.name,
+                    features=self.features,
+                    overwrite_trainee=overwrite,
+                    persistence=self.persistence,
+                    id=self.id,
                     library_type=library_type,
                     max_wait_time=max_wait_time,
-                    overwrite_trainee=overwrite,
-                    resources=resources
+                    project=self.project,
```

Comment:
Shouldn't pass engine instances to the underlying clients. Instead pass the project id.

Reply:
Addressed by: ae85d46